quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusCompletions.java

```diff
@@ -209,5 +212,73 @@ private static String getPropertyName(String propertyName, boolean snippetsSuppo
 	private static void collectPropertyValueSuggestions(Node node, QuarkusProjectInfo projectInfo,
 			QuarkusCompletionSettings completionSettings, CompletionList list) {
 
+		if (node.getParent() != null &&
+			node.getParent().getNodeType() != Node.NodeType.PROPERTY) {
+			return;
+		}
+
+		PropertyKey key = getPropertyKeyFromProperty((Property) node.getParent());
+
+		if (key == null) {
+			return;
+		}
+
+		String keyText = key.getText().trim();
+
+		ExtendedConfigDescriptionBuildItem property = 

```

Comment:
replace with `ExtendedConfigDescriptionBuildItem property = projectInfo.getProperty(keyText);`

It will support `profil and map cases` when 
https://github.com/redhat-developer/quarkus-ls/issues/54 will be implemented.